src/main/java/in/ac/bits/protocolanalyzer/persistence/repository/ElasticSearchConfig.java

```diff
@@ -0,0 +1,33 @@
+package in.ac.bits.protocolanalyzer.persistence.repository;
+
+import org.elasticsearch.common.settings.ImmutableSettings;
+import org.elasticsearch.node.NodeBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.data.elasticsearch.core.ElasticsearchOperations;
+import org.springframework.data.elasticsearch.core.ElasticsearchTemplate;
+import org.springframework.data.elasticsearch.repository.config.EnableElasticsearchRepositories;
+
+@Configuration
+@EnableElasticsearchRepositories(basePackages = 
+		"in.ac.bits.protocolanalyzer.persistence.repository")
+public class ElasticSearchConfig {
+    
+	@Autowired
```

Comment:
@adityaketkar The line indentation is not uniform across the file. Please adjust.

Reply:
@prasadtalasila the newline is to prevent the sentence from crossing the 100 character limit imposed by codeclimate